src/main/java/net/pms/dlna/WebStream.java

```diff
@@ -92,7 +92,9 @@ public DLNAThumbnailInputStream getThumbnailInputStream() throws IOException {
 				FileUtil.isUrl(thumbURL) ? downloadAndSend(thumbURL, true) : new FileInputStream(thumbURL)
 			);
 		} else {
-			return super.getThumbnailInputStream();
+			try (DLNAThumbnailInputStream dtis = super.getThumbnailInputStream()) {
+				return dtis;
```

Comment:
@SubJunk there are Eclipse warnings `Resource leak: '<unassigned Closeable value>' is not closed at this location` so I made this change. It works for me. I tested other way and it could be written like 

> DLNAThumbnailInputStream dtis = super.getThumbnailInputStream();
> 			return dtis;

Both works and the the Eclipse warnings are eliminated.

Reply:
I don't see how it can still do what it did before, since now it is returning a closed stream. How can the consumer use a closed stream? As I said in https://github.com/UniversalMediaServer/UniversalMediaServer/pull/2066#issuecomment-636412907 I think we should instead make sure that the consumers close the streams they receive.
I have seen recommendations for naming conventions too, where any method that returns a closeable stream should be prefixed with `output` or something, and then we can add a note to the JavaDoc for those methods to make sure. Something like:

```
/**
 * Method description.
 *
 * Note: If you use this method, you must close the resource you receive after you are finished with it.
 */